README.md

```diff
@@ -162,6 +162,22 @@ $ ruby start_finish_trace.rb
 
 **Busted** includes an [example `dtrace` probe](/dtrace/probes/examples/method-cache-clear.d) for use on the command line or an application.  See the [probe](/dtrace/probes/examples/method-cache-clear.d) for usage.
 
+## Rails Usage
+
+In rails you can use the `BustedTracer` middleware to log the method and constant cache invalidations. You can configure BustedTracer using,
+
+```ruby
+config.middleware.insert_after(ActionDispatch::Static, BustedTracer)
```

Comment:
Why don't we test `BustedTracer ` comes before the rest of the middleware stack:


```
config.middleware.insert_before(0, BustedTracer)
```

Reply:
@HParker My thoughts are `config.middleware.insert_before 0` makes sure `BustedTracer` runs at the beginning of the stack.

Why did you suggest `ActionDispatch::Static`?  I've got a Rails 5.0 app handy, and here's its out-of-the-box middleware stack.

```
~/Code/rails_5
❯ rails --version
Rails 5.0.1

~/Code/rails_5
❯ rake middleware
Running via Spring preloader in process 63804
use Rack::Sendfile
use ActionDispatch::Static
use ActionDispatch::Executor
use ActiveSupport::Cache::Strategy::LocalCache::Middleware
use Rack::Runtime
use Rack::MethodOverride
use ActionDispatch::RequestId
use Sprockets::Rails::QuietAssets
use Rails::Rack::Logger
use ActionDispatch::ShowExceptions
use WebConsole::Middleware
use ActionDispatch::DebugExceptions
use ActionDispatch::RemoteIp
use ActionDispatch::Reloader
use ActionDispatch::Callbacks
use ActiveRecord::Migration::CheckPending
use ActionDispatch::Cookies
use ActionDispatch::Session::CookieStore
use ActionDispatch::Flash
use Rack::Head
use Rack::ConditionalGet
use Rack::ETag
run Rails5::Application.routes
```